features/admin_features/admin_adds_new_user.feature

```diff
@@ -5,10 +5,66 @@ Feature: Admin can administer users
 
   Background:
     Given An admin exists "admin@example.com" "password"
+    Given An admin exists "restaurant_owner@example.com" "password" "false"
     And I am logged in as admin
     And I go to the dashboard
 
   Scenario: Add new user
     Given I click "Users"
     Then I click "New User"
-    And I fill in "Email" with "user@example.com"
\ No newline at end of file
+    And I fill in "Email" with "user@example.com"
+
+  Scenario: Add new admin owner
+    Given I click "Admin Users"
+    Then I click "New Admin User"
+    And I fill in "Email" with "admin2@example.com"
+    And I fill in "Password*" with "password"
+    And I fill in "Password confirmation" with "password"
+    And I click_on_checkbox "Admin"
+    And I click "Create Admin user"
+    Then I should see "Admin user was successfully created."
```

Comment:
Okay, what kind of user will be created with this flow? It does not come across easily.